lib/components/map/map.js

```diff
@@ -14,7 +14,7 @@ class Map extends Component {
     }
   }
 
-  getComponentForView (view) {
+  getComponentForView = view => {
```

Comment:
Why this change? Since there is no access to instance variables, I don't think this is needed.

Reply:
Reverted in https://github.com/opentripplanner/otp-react-redux/pull/339/commits/794aeafcdf59f7ea394a00aac565daa4e7a9b2f0.